check/controls.go

```diff
@@ -185,6 +200,69 @@ func (controls *Controls) JUnit() ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// AASF encodes the results of last run to AWS Security Finding Format(AASF).
+func (controls *Controls) AASF() ([]*securityhub.AwsSecurityFinding, error) {
```

Comment:
```suggestion
func (controls *Controls) ASFF() ([]*securityhub.AwsSecurityFinding, error) {
```

Reply:
this typo in the method name would need corecting elsewhere too, I am not sure I will catch them all